### PR TITLE
Cadl Website 

### PR DESCRIPTION
**Pull request by timotheeguerin**
cleanup and improvement over the initial website implementation in #875.
Cleanup UI pieces that we don't need(at least for now). 
- [x] Tweak the colors a little.
- [x]  Setup docs navigation
- [x]  Setup syntax highlighting for code snippet
   - [x]  Including cadl: Better than github docs now!
- [x] Publish to storage account 
image

### CODE REVIEW

File: packages/website/cadl-prism-lang.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  keyword:
+    /\b(?:import|model|namespace|op|interface|union|using|is|extends|enum|alias|return|void|never|if|else|projection)\b/,
+
+  function: /\b[a-z_]\w*(?=[ \t]*\()/i,
+
+  number: /(?:\b\d+(?:\.\d*)?|\B\.\d+)(?:E[+-]?\d+)?/i,
+  operator:
+    /--|\+\+|\*\*=?|=>|&&=?|\|\|=?|[!=]==|<<=?|>>>?=?|[-+*/%&|^!=<>]=?|\.{3}|\?\?=?|\?\.?|[~:]/,
+  punctuation: /[{}[\];(),.:]/,
+};
```
Comment (nguerrera) [inline packages/website/cadl-prism-lang.js:58]: I wonder if we can use the tmlanguage somehow? Seems annoying to have another set of regexes to maintain.

Comment (timotheeguerin) [reply packages/website/cadl-prism-lang.js:58]: Haha, yeah..., I did look if there was an existing way to auto convert it but doesn't seem like there is. Felt more work to get this to work.

Comment (nguerrera) [reply packages/website/cadl-prism-lang.js:58]: Maybe file an issue?

Comment (timotheeguerin) [reply packages/website/cadl-prism-lang.js:58]: oh you mean we are the one doing the syntax highlighting instead of using the extension

Comment (nguerrera) [reply packages/website/cadl-prism-lang.js:58]: Maybe? Just brainstorming here.

Comment (timotheeguerin) [reply packages/website/cadl-prism-lang.js:58]: Created an issue https://github.com/microsoft/cadl/issues/938

This was the PR that added bicep to PrismJS, where they seemed to have manually intepreted the tmlanguage and wrong a simple primsjs language
https://github.com/PrismJS/prism/pull/3027

**Review by nguerrera — APPROVED** [review]

**Pull request merged**